trunk/src/modules/paraview.F90

```diff
@@ -50,7 +58,8 @@ end subroutine paraview_init
 !> @date Feb 2023
 !-----------------------------------------------------------------------------
    subroutine paraview_finalize()
-      call hdf5_w_finalize()
+      !only call hdf5_w_finalize when xdmf output is chosen
+      if(VIS_VTU .eqv. .false.) call hdf5_w_finalize()
```

Comment:
Make style consistent with other parts of the code
```suggestion
!     only call hdf5_w_finalize when xdmf output is chosen
      if (.not. VIS_VTU) call hdf5_w_finalize()
```

Reply:
Additionally, `paraview_init` is still being called which initializes HDF5's file pointers?

Reply:
Change implemented